fastlane/lib/fastlane/actions/create_keychain.rb

```diff
@@ -25,6 +26,7 @@ def self.run(params)
 
         if !exists?(keychain_path)
           commands << Fastlane::Actions.sh("security create-keychain -p #{escaped_password} #{keychain_path}", log: false)
+          Actions.lane_context[Actions::SharedValues::KEYCHAIN_PATH] = keychain_path
```

Comment:
Do we want to bump this outside and above of this if statement? I think it would be good to set the lane context even if the keychain already exists 😊 Thoughts?

Reply:
yes of course!